Gaijinizer patch/Map089.txt

```diff
@@ -4,137 +4,158 @@
-I came well, \\N[1]. I was waiting for you.#MTLed
+I'm glad you're here, \\N[1].
+
+I was waiting for you.#Strate
 > END STRING
 
 > BEGIN STRING
 ！！！ああああっ♡
 
 る、ルシィ…！？
 > CONTEXT: Map089/events/5/pages/0/30/Dialogue/kazuya-0
-!!! AAAAh ♡. Ru, Lucy ...! ?#MTLed
+!!! AAAAH♡
+
+L-Lucy...!?#Strate
 > END STRING
 
 > BEGIN STRING
 『あんな事』に私の手を汚させておいて……
 
 よくもおめおめとここを訪れましたね？
 > CONTEXT: Map089/events/5/pages/0/34/Dialogue/yuuma
-Make my hands dirty in "an such thing" ...
-Did you visit here with disgrace well?#MTLed
+After dirtying my hands with "that thing"......
+
+You dare to visit me so shamelessly?#Strate
 > END STRING
 
 > BEGIN STRING
 うああああああっ！！♡♡やめっ……！！♡
 
 ちがっ…！！♡誤解っ……！！♡なんっ…！！
 > CONTEXT: Map089/events/5/pages/0/43/Dialogue/kazuya-0
-whaaaaah! Stop it ...!!Moth. 
-Different ...!!Misunderstanding......! !
-Cunning umph ...! !#MTLed
+Uaaaaaaah!! ♡♡Stop...!!♡ 
+
+Wrong...!! ♡Misunderstanding......!! Wha...!!#Strate
 > END STRING
 
 > BEGIN STRING
 あひいいいいいいいいいいぃぃぃぃ！！♡♡
 
 それはやめてええええええ！！♡♡
 > CONTEXT: Map089/events/5/pages/0/52/Dialogue/kazuya-0
-Oh nice nice nice nice nice nooo! Moth
-That's stop yeah yeah yeah!!Moth#MTLed
+Ahiiiiiiiiiiiiii!! ♡♡
+
+Stop iiiiiiiiiiiiiiit!!♡♡#Strate
 > END STRING
 
 > BEGIN STRING
 あれから、どんなことをされたのですか…？
 
 包むことに関しては、私も負けていませんが…
 > CONTEXT: Map089/events/5/pages/0/56/Dialogue/yuuma
-What did you do from that ...?
-As for wrapping, I have not lost ...#MTLed
+What did they do to you after that...?
+
+When it comes to wrapping, I can't be beaten...#Strate
 > END STRING
 
 > BEGIN STRING
 あなたがあの小娘どもから、された量の百倍くらいは…
 
 私に搾られてくださいね…♡
 > CONTEXT: Map089/events/5/pages/0/60/Dialogue/yuuma
-About a hundred times of the amount you got from
-a little girl...Please squeeze me...♡#MTLed
+Please give me...about a hundred times the amount
+that those little girls squeezed out of you.♡#StrateLocalization
 > END STRING
 
 > BEGIN STRING
 ああああああああああああああ！！♡♡
 
 たひゅけてえええええええええ！！♡♡
 > CONTEXT: Map089/events/5/pages/0/69/Dialogue/kazuya-0
-Ah ah ah ah ah ah ah!!Moth. 
-Oh yeah yeah yeah yeah!!Moth#MTLed
+AAAAAAAAAAAAAAH!!♡♡
+
+S-SAVE MEEEEEEEEE!!♡♡#Strate
 > END STRING
 
 > BEGIN STRING
 ……………連れて参りました。
 
 我が主よ。
 > CONTEXT: Map089/events/5/pages/0/82/Dialogue/other1
-...... I took it. My Lord.#MTLed
+.........I brought him.
+
+My Lord.#Strate
 > END STRING
 
 > BEGIN STRING
 うわっ
 
 なんだそれ
 > CONTEXT: Map089/events/5/pages/0/86/Dialogue/クッション君
-Wow. what is that#MTLed
+Wow.
+
+What is that?#Strate
 > END STRING
 
 > BEGIN STRING
 \\N[1]様でございます。
 
 ………少々、きつく搾られてしまったようですね。
 > CONTEXT: Map089/events/5/pages/0/90/Dialogue/other1
-It is Mr. \\N[1]....It seems that you have been
-squeezed a little bit tight.#MTLed
+It is \\N[1]-sama.
+
+It appears he was squeezed quite tightly.#Strate
 > END STRING
 
 > BEGIN STRING
 すげぇなぁ
 
 そんな干からび方、マンガでしか見たことねぇぞ。
 > CONTEXT: Map089/events/5/pages/0/94/Dialogue/クッション君
-I'm sorry. I've seen it only in manga if you like
-it.#MTLed
+Amazing.
+
+I've only seen such a way of drying out in manga.#Strate
 > END STRING
```

Comment:
Light rewording of the second sentence for flow:
_**I've only seen people dried up like that in manga.**_